<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stronghold</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/shield.png</normaloff>../res/shield.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="boxStatus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>800</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="uiTable" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>641</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboTable">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>251</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAdd">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>39</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="insert-table-row">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRemove">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>39</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="delete-table-row">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSwitch">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="view-refresh">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCommit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>39</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="document-save">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGenerate">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>39</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="password-copy">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCreate">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>39</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="insert-table">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionImport_Table"/>
    <addaction name="actionImport_Database"/>
    <addaction name="actionExport_Table"/>
    <addaction name="actionExport_Database"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_XSoftware"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionImport_Table">
   <property name="text">
    <string>&amp;Import Table</string>
   </property>
  </action>
  <action name="actionImport_Database">
   <property name="text">
    <string>Import &amp;Database</string>
   </property>
  </action>
  <action name="actionExport_Table">
   <property name="text">
    <string>&amp;Export Table</string>
   </property>
  </action>
  <action name="actionExport_Database">
   <property name="text">
    <string>E&amp;xport Database</string>
   </property>
  </action>
  <action name="actionGenerator">
   <property name="text">
    <string>&amp;Generator</string>
   </property>
  </action>
  <action name="actionAbout_XSoftware">
   <property name="text">
    <string>&amp;About XSoftware</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About &amp;Qt Framework</string>
   </property>
  </action>
  <action name="actionAdminMode">
   <property name="text">
    <string>&amp;Switch Admin Mode</string>
   </property>
  </action>
  <action name="actionDatabaseMode">
   <property name="text">
    <string>Switch &amp;Database Mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>uiTable</class>
   <extends>QTableView</extends>
   <header>uitable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
